xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/v3/service/OkexAccountServiceRaw.java

```diff
@@ -63,7 +85,8 @@ public OkexWithdrawalResponse withdrawal(OkexWithdrawalRequest req) throws IOExc
     FuturesPositionsResponse res =
         okex.getFuturesPositions(apikey, digest, timestamp(), passphrase);
     res.checkResult();
-    return res.getHolding().get(0);
+    List<List<FuturesPosition>> holding = res.getHolding();
+    return holding == null || holding.isEmpty() ? new ArrayList<>() : holding.get(0);
```

Comment:
`Collections.EMPTY_LIST` instead of `new ArrayList<>()` would be better

Reply:
you mean Collections.emptyList() :-)